pycqed/analysis/measurement_analysis.py

```diff
@@ -3819,6 +3823,57 @@ def run_default_analysis(self, print_fit_results=False,
         return fit_res
 
 
+class Acquisition_Delay_Analysis(MeasurementAnalysis):
```

Comment:
I assume this is a default analysis for a new experiment (we always did this by hand). It looks quite useful :+1: .